core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/package_replace_options_repository.go

```diff
@@ -0,0 +1,89 @@
+package git_package_content_provider
```

Comment:
hey! how do we decide to keep it here instead of `enclave_db` - `container-engine-lib/lib/database_accessors`

Reply:
because it's just APIC scope, but we could discuss this if there is a previous definition for it?